test/unit/attributes/initializer.spec.js

```diff
@@ -1,9 +1,20 @@
 const { expect } = require('chai');
+const { attributes } = require('../../../src');
 const Initializer = require('../../../src/attributes/initializer');
 
 describe('Initializer', () => {
   let attributeDescriptor, instance;
 
```

Comment:
Do you think it's possible to add a test to ensure we're coercing a single time? I'm just rambling about it.

Reply:
I'll try to create a `spy` on `Descriptor#coerceAttributes` function to ensure it was called once.